Formula/tcpreplay.rb

```diff
@@ -12,23 +12,19 @@ class Tcpreplay < Formula
     sha256 "ab0379428462fbdc2653feae2bcc404cf6b6d2129ddf0461019c53794ce87e4f" => :mavericks
   end
 
-  depends_on "libdnet" => :recommended
+  depends_on :xcode
```

Comment:
@DomT4 OK, looks like this works fine since it's trying to set isysroot:
```
    if MacOS::Xcode.installed?
      args << "--with-macosx-sdk=#{MacOS.sdk.version}"
    else
      inreplace "configure",
        /^.*Could not figure out the location of a Mac OS X SDK.*$/,
        "MACOSX_SDK_PATH=/"
    end
```